autopush/tests/test_websocket.py

```diff
@@ -31,6 +31,28 @@
 from autopush.utils import base64url_encode, ms_time
 
 
+dummy_version = u'gAAAAABX_pXhN22H-hvscOHsMulKvtC0hKJimrZivbgQPFB3sQAtOPmb' \
```

Comment:
fyi (don't care about changing it) you can surround these in parens to get the auto concatenate w/out end of the line \s.

or a = u"""\<newline>gAA<...>OX8=""".replace("\n", "") works pretty nice for blocks like these